eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -1,4 +1,5 @@
 <Project>
+  <Import Project="$(ToolsILLinkDir)build/Microsoft.NET.ILLink.Tasks.props" />
```

Comment:
That file is expected to imported before the project body is evaluated. Please move this into a Directory.Build.props file so that we build the same as customers do via the PackageReference.